### PR TITLE
PAI | 更新脱毛相关信息

### PR DESCRIPTION
**Pull request by AnnaBeiyan**
信息来源：https://pai.co.th/service/penile-skin-inversion-scrotal-skin-graft-male-to-female-trans-women/

> Q: What is the average wait-list for penile inversion or other method at PAI?
> A: Schedule/planning at least 1 month ahead of time.
>
> Q: Is genital hair removal mandatory for penile inversion?
> A: No. (this is not necessary but just an option). For your sake, even thru, the surgeon will also scrape the root of hair from penile and scrotal skin during surgical times (hair follicle scraping). 

relate #617

### CODE REVIEW

File: content/zh-cn/docs/srs/thailand/preecha.md [hunk 1]
```diff
@@ -134,6 +134,7 @@ HIV 阳性需额外加收 50% 费用，CD4 必须超过 500 且病毒载量未
 1. 尿道成形术
 1. 阴唇成形术。主要和次要龟头（阴茎头）的一部分，连同其神经和血管，被转化为阴蒂。
 1. **尿道周围多余的勃起组织将去除**，以避免在性唤起期间勃起组织充血引起的症状。
+1. 医生会在手术时刮掉部分区域皮肤的发根，以避免术后阴道内生长毛发。因此手术前进行生殖器位置脱毛不是必须的。
```
Comment (LittleSound) [inline content/zh-cn/docs/srs/thailand/preecha.md:137]: 与我了解的情况类似，另外医生说构造阴道内部使用到的那部分皮肤一般情况下就不会有什么毛。

Comment (AnnaBeiyan) [reply content/zh-cn/docs/srs/thailand/preecha.md:137]: 那这部分需要修改吗？

Comment (LittleSound) [reply content/zh-cn/docs/srs/thailand/preecha.md:137]: 这不是修改建议，你可以自己决定。

File: content/zh-cn/docs/srs/thailand/preecha.md [hunk 2]
```diff
@@ -134,6 +133,7 @@ HIV 阳性需额外加收 50% 费用，CD4 必须超过 500 且病毒载量未
...
```
Comment (Cryolitia) [inline content/zh-cn/docs/srs/thailand/preecha.md:136]: 这里加个引用来源

**Review by Cryolitia — CHANGES_REQUESTED** [review]

**Comment by Cryolitia** [comment]
另外繁体中文我建议直接删，不要机翻，没有维护者就让他们看简体好了，CC @project-trans/chairs

File: content/zh-cn/docs/srs/thailand/preecha.md (hunk 2 above)
Comment (AnnaBeiyan) [reply content/zh-cn/docs/srs/thailand/preecha.md]: 加了

**Review by Cryolitia — APPROVED** [review]

**Review by BeiyanYunyi — APPROVED** [review]

**Review by LittleSound — APPROVED** [review]

**Pull request merged**